druid/examples/game_of_life.rs

```diff
@@ -12,125 +12,35 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#![allow(clippy::unreadable_literal)]
-
-//! Game of life
+//! This is an example of how you would implement the game of life with druid.
+//! This example doesnt showcase anything specific in druid.
 
 use std::ops::{Index, IndexMut};
 use std::time::{Duration, Instant};
 
 use druid::widget::prelude::*;
 use druid::widget::{Button, Flex, Label, Slider};
 use druid::{
-    AppLauncher, Color, Data, Lens, LocalizedString, MouseButton, Point, Rect, TimerToken,
-    WidgetExt, WindowDesc,
+    AppLauncher, Color, Data, Lens, MouseButton, Point, Rect, TimerToken, WidgetExt, WindowDesc,
 };
 use std::sync::Arc;
 
-const GRID_SIZE: usize = 40;
+const GRID_SIZE: usize = 41;
 const POOL_SIZE: usize = GRID_SIZE * GRID_SIZE;
 
 const BG: Color = Color::grey8(23 as u8);
-const C0: Color = Color::from_rgba32_u32(0xEBF1F7);
-const C1: Color = Color::from_rgba32_u32(0xA3FCF7);
-const C2: Color = Color::from_rgba32_u32(0xA2E3D8);
-const C3: Color = Color::from_rgba32_u32(0xF2E6F1);
-const C4: Color = Color::from_rgba32_u32(0xE0AFAF);
+const C0: Color = Color::from_rgba32_u32(0xEBF1F7); //Color::rgb(235, 241, 247)
+const C1: Color = Color::from_rgba32_u32(0xA3FCF7); //Color::rgb(162,252,247)
+const C2: Color = Color::from_rgba32_u32(0xA2E3D8); //Color::rgb(162,227,216)
+const C3: Color = Color::from_rgba32_u32(0xF2E6F1); //Color::rgb(242,230,241)
+const C4: Color = Color::from_rgba32_u32(0xE0AFAF); //Color::rgb(224,175,175)
```

Comment:
I agree that using `Color::rgb` would be better here.

Reply:
yup, but sadly that isnt possible. I made them already but they're not const so decided to put them as comments.

Reply:
Oh right I meant `Color::rgb8`. That's the `const` one that takes u8s:

```rust
const C0: Color = Color::rgb8(0xEB, 0xF1, 0xF7); 
// same as:
const C0: Color = Color::from_rgba32_u32(0xEBF1F7);
```

Reply:
I would also actually fix up these names while we're in here, like `BG` could be `BACKGROUND` and then I'd try and give more meaningful names to the other colors too?

Reply:
I just made it an array, much simpler.